### PR TITLE
Bringing the bootstrapped environment up

### LINKED ISSUE #120: Bringing the bootstrapped environment up in CI
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

Add e2e test script for https://github.com/redhat-developer/kam/tree/master/docs/journey/day1#bringing-the-bootstrapped-environment-up in [feature](https://github.com/redhat-developer/kam/blob/master/test/e2e/features/basic.feature) file.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by amitkrout**
**What type of PR is this?**
/kind enhancement

**What does this PR do / why we need it**:
https://github.com/redhat-developer/kam/issues/120
**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #120 

**How to test changes / Special notes to the reviewer**:
Test scenario should pass in CI

### CODE REVIEW

File: test/e2e/kamsuite/kamsuite.go [hunk 1]
```diff
@@ -142,3 +143,30 @@ func deleteGithubRepository(repoURL, token string) {
 		log.Printf("Successfully deleted repository: %q", repoURL)
 	}
 }
+
+func createRepositry() error {
```
Comment (jgwest) [inline test/e2e/kamsuite/kamsuite.go:147]: Typo in the function name

File: test/e2e/features/basic.feature [hunk 2]
```diff
@@ -18,3 +18,14 @@ Feature: Basic test
     Scenario: KAM bootstrap command should fail if any one mandatory flag --git-host-access-token is missing
         When executing "kam bootstrap --service-repo-url $SERVICE_REPO_URL --gitops-repo-url $GITOPS_REPO_URL" fails
         Then exitcode should not equal "0"
+
+    Scenario: Bringing the bootstrapped environment up
+        Given gitops repositry is created
```
Comment (jgwest) [inline test/e2e/features/basic.feature:23]: Typo in repository

File: test/e2e/features/basic.feature [hunk 3]
```diff
@@ -18,3 +18,14 @@ Feature: Basic test
...
+    Scenario: Bringing the bootstrapped environment up
+        Given gitops repository is created
+        When executing "kam bootstrap --service-repo-url $SERVICE_REPO_URL --gitops-repo-url $GITOPS_REPO_URL --image-repo $IMAGE_REPO --dockercfgjson $DOCKERCONFIGJSON_PATH --git-host-access-token $GIT_ACCESS_TOKEN --output bootstrapresources --overwrite" succeeds
+        Then executing "cd bootstrapresources" succeeds
+        Then executing "git init ." succeeds
+        Then executing "git add ." succeeds
+        Then executing "git commit -m 'Initial commit.'" succeeds
+        Then executing "git remote add origin $GITOPS_REPO_URL" succeeds
+        Then executing "git branch -M main" succeeds
+        Then executing "git push -u origin main" succeeds
```
[the remark this answers is not in the document]
Comment (varshab1210) [reply test/e2e/features/basic.feature:31]: I have created a pr https://github.com/redhat-developer/kam/pull/235. 

> Also we need to mention the creation of a private gitops repo before executing these steps.

I can see the creation of private gitops repo in the prereq of the bootstrapped environment up steps - https://github.com/redhat-developer/kam/tree/master/docs/journey/day1#bringing-the-bootstrapped-environment-up

**Pull request merged**